module/projectAttributes/helper.js

```diff
@@ -0,0 +1,210 @@
+const projectAttributesQueries = require(DB_QUERY_BASE_PATH + '/projectAttributes')
+const entitiesService = require(GENERICS_FILES_PATH + '/services/entity-management')
+
+module.exports = class ProjectAttributesHelper {
+	/**
+	 * create project attributes
+	 * @method
+	 * @name create
+	 * @returns {Object} .
+	 */
+	static async create() {
+		try {
+			// Get the default project attributes
+			let getProjectAttribute = CONSTANTS.common.DEFAULT_ATTRIBUTES
+			//Getting roles from the entity service
+			let userRoleInformation = await entitiesService.userExtensionDocuments(
+				{
+					status: CONSTANTS.common.ACTIVE_STATUS.toUpperCase(),
+				},
+				['title', 'code']
+			)
+
+			if (!userRoleInformation.success) {
+				throw new Error(userRoleInformation)
+			}
+			let roleToAddForFilter = []
+			if (userRoleInformation.data.length > 0) {
+				userRoleInformation.data.map((eachResult) => {
+					roleToAddForFilter.push({
+						label: eachResult.title,
+						value: eachResult.code,
+					})
+				})
+			}
+			// Adding the roles into entities of projectAttributes
+			getProjectAttribute[1].entities = roleToAddForFilter
+			await projectAttributesQueries.createProjectAttributes(getProjectAttribute)
+			return {
+				success: true,
+				message: CONSTANTS.apiResponses.PROJECT_ATTRIBUTES_CREATED,
+			}
+		} catch (error) {
+			// If an error occurs, return an error response with status, message, and the error object
+			return {
+				status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+				message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+				errorObject: error,
+			}
+		}
+	}
+
+	/**
+	 * update project attributes
+	 * @method
+	 * @name update
+	 * @param {codeToUpdate} - which attributes need to be updated
+	 * @param {languageCode} - language code for multilingual
+	 * @param {Object} translateData  - TranslatedData object(reqBody)
+	 * @returns {Object} .
+	 */
+
+	static async update(codeToUpdate, languageCode, translateData) {
```

Comment:
@praveenKDass rename param to code and language

Reply:
Changed @VISHNUDAS-tunerlabs